RELEASE/scripts/postsool.ash

```diff
@@ -731,6 +738,11 @@ void handlePostAdventure()
 		buffMaintain($effect[Singer\'s Faithful Ocelot], 280, 1, 10);
 		if(doML)
 		{
+			// Catch when we leave Smut Orc
+			if(0 < have_effect($effect[Driving Intimidatingly]))
+			{
+				uneffect(Driving Intimidatingly);
+			}
```

Comment:
I think this should go in presool as well.

Also I'm worried about cases like "we're doing the chasm and need the -ML, then move somewhere else to burn delay, then return to the chasm immediately afterwards". I think it's incorrect to remove Driving Intimidatingly in that case, since you'll just reapply it on your return.

The two choices I see are to
1) not uneffect Driving Intimidatingly, since it'll get overridden by the next asdonmartin use anyway
2) keep track of the last turn you had Driving Intimidatingly and wanted to have Driving Intimidatingly, and if it's been more than two turns since you wanted it and you don't want it now, uneffect it in `presool`

Reply:
Good point on the delay. I added it in my latest push.